targets/esp8266/jshardware.c

```diff
@@ -618,9 +657,10 @@ void jshSPISend16(
     IOEventFlags device, //!< Unknown
     int data             //!< Unknown
   ) {
-  os_printf("ESP8266: jshSPISend16\n");
+  os_printf("> jshSPISend16 - device=%d, data=%x\n", device, data);
```

Comment:
These won't be helping with the speed *at all*, and you have `spi_tx16` available - so use that instead. It should make a big difference.